Standards/scs-0302-v2-domain-manager-role.md

```diff
@@ -0,0 +1,143 @@
+---
+title: Domain Manager adoption notes
+type: Standard
+track: IAM
+status: Draft
+replaces: scs-0302-v1-domain-manager-role.md
+---
+
+## Introduction
+
+After the domain manager persona has been implemented certain issues in the
+standard adoption and verification started being raised by CSPs:
+
+- Not every CSP is using domains to separate customers.
+
+- CSP may rely on the Identity federation in which case it is impossible or is
+prohibited to manage identities on OpenStack side (OpenStack is a service
+provider and not an identity provider).
+
+- CSP may customize authorization policies in a different way so that domain
+manager can not be implemented by simply reusing the upstream implementation.
+
+As such simple enforcement of the Domain Manager persona can not be achieved.
+
+This standard clarifies base standard and splits requirements into recommended
+and mandatory to provide better granularity while still giving guidance with
+the goal to provide a smooth user experience for the end users.
+
+Requiring customer to use CSP specific APIs to manage identity data is
+contradicting with the idea of certification as such. It hinders customers from
```

Comment:
Somehow in my brain this sentence works better with standardization (since that comes before certification, which is just the attestation of successful standardization):

```suggestion
Requiring customer to use CSP specific APIs to manage identity data is
contradicting the idea of standardization  as such. It hinders customers from
```